business/enums.py

```diff
@@ -22,6 +23,7 @@ class OfferType(enum.IntEnum):
     OfferType.PART_TIME: "Part-time job",
     OfferType.FULL_TIME: "Full-time job",
     OfferType.VOLUNTEER: "Volunteering",
+    OfferType.MASTER_THESIS: "Master Thesis",
```

Comment:
Nitpicking, no need to capitalize like in American/British locale (so "Master thesis" instead).